clients/src/main/resources/common/message/FetchSnapshotRequest.json

```diff
@@ -18,7 +18,7 @@
   "type": "request",
   "listeners": ["controller"],
   "name": "FetchSnapshotRequest",
-  "validVersions": "0",
+  "validVersions": "0-1",
```

Comment:
Please add a comment to mention what changed in v.1.

Reply:
Fixed.